project_templates/roundtable_aiohttp_bot/{{cookiecutter.repo_name}}/src/{{cookiecutter.package_name}}/handlers/external/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""Externally-accessible endpoint handlers that serve relative to
+``/<app-name>/``.
+"""
+
+__all__ = ["get_index"]
+
+from .index import get_index
```

Comment:
You're using absolute imports everywhere else (thank you!). I think they should be used here too.